project.godot

```diff
@@ -74,4 +74,5 @@ common/enable_pause_aware_picking=true
 [rendering]
 
 quality/driver/driver_name="GLES2"
+2d/snapping/use_gpu_pixel_snap=true
```

Comment:
I keep seeing this changing - not sure which option should be correct

Reply:
maybe i changed it? this is one of the things we need to decide on. do we want pixel snapping or not? idk why it's GPU but godot used to have different settings for pixel snapping